holmes/plugins/toolsets/aks.yaml

```diff
@@ -69,3 +69,13 @@ toolsets:
         user_description: "get resource group for cluster {{ CLUSTER_NAME }}"
         command: |
            az aks list --subscription {{ SUBSCRIPTION_ID }} --only-show-errors --query "[?name=='{{ CLUSTER_NAME }}'].resourceGroup | [0]" -o tsv
+      - name: "get_node_resource_group"
+        description: "Retrieves the node resource group name for the AKS cluster"
+        user_description: "get the node resource group name for cluster {{ CLUSTER_NAME }}"
+        command: |
+          az aks list --subscription {{ SUBSCRIPTION_ID }} --only-show-errors --query "[?name=='{{ CLUSTER_NAME }}'].nodeResourceGroup | [0]" -o tsv
+      - name: "get_api_server_public_ip"
+        description: "get the public IP of kube-apiserver for a public AKS cluster"
+        user_description: "get the public IP of kube-apiserver for a public AS cluster {{ CLUSTER_NAME }}"
```

Comment:
Does the description works properly here, with the `CLUSTER_NAME`? (because it's not part of the tool paramters)